index.js

```diff
@@ -125,6 +125,9 @@ module.exports = {
 						selector: 'memberLike',
 						modifiers: ['private'],
 						format: ['camelCase'],
+						// Support underscores in property names
+						leadingUnderscore: 'allow',
```

Comment:
This doesn't allow sth like `foo_bar_baz`. I think you want instead to add `snake_case` to the `format` field (line 127).